config/locales/fr.yml

```diff
@@ -1119,6 +1120,34 @@ fr:
       peer_visible: "Rendre visible à leurs auteurs pairs"
       visibility_error: "Sélectionnez au moins une case dans la section de visibilité"
 
+    checkbox_criteria:
+      title: "Critères d' Checkbox"
+      move_up: "Monter"
```

Comment:
Most of these strings are shared among criteria. Perhaps, we can put them under criteria which is immediately above.